tuftool/README.md

```diff
@@ -115,8 +115,29 @@ tuftool update \
    --timestamp-version 2 \
    --outdir "${WRK}/tuf-repo" \
    --metadata-url file:///$WRK/tuf-repo/metadata
+
+#[Optional] Set an RHTAS target (fulcio, ctlog, rekor, tsa)!
+touch "${WRK}/input/ctfe.pub" 
+
+tuftool rhtas \
```

Comment:
When I run this command with a new input in the `${WRK}/input/` directory, the new input isn't copied to the `${WRK}/tuf-repo/targets/` directory. Am I doing something wrong?

Reply:
Great catch!